packages/react-performance/README.md

```diff
@@ -17,7 +17,36 @@ $ yarn add @shopify/react-performance
 
 The most basic way to use the tools in this package is to record information and display it locally to the user. In practice you usually only want to do this in development so that developers can easily see performance information.
 
-#### Display data using NavigationListener
+### Adding some `<PerformanceMark />`s
+
+We will need to add some `<PerformanceMark />` components to our React application in order to help `@shopify/react-performance` determine when your application is fully loaded and React has mounted your components.
```

Comment:
```suggestion
We will need to add some `<PerformanceMark stage="complete"/>` components to our React application in order to help `@shopify/react-performance` determine when your application is fully loaded and React has mounted your components.
```

Reply:
I suggest this since `when your application is fully loaded and React has mounted your components` is only true with `stage="complete"`